pkg/scalers/azure_servicebus_scaler.go

```diff
@@ -102,8 +102,8 @@ func parseAzureServiceBusMetadata(config *ScalerConfig) (*azureServiceBusMetadat
 	if meta.entityType == none {
 		return nil, fmt.Errorf("no service bus entity type set")
 	}
-
-	if config.PodIdentity == "" || config.PodIdentity == kedav1alpha1.PodIdentityProviderNone {
+	switch {
```

Comment:
can we rewrite this switch to actually switch over config.PodIdentity?

Reply:
I will do this.